src/js/apps/patients/patients-main_app.js

```diff
@@ -75,18 +76,19 @@ export default RouterApp.extend({
     this.startRoute('patient', { patientId });
   },
 
-  showPatientsWorklist(worklistId) {
+  showPatientsWorklist(worklistId, worklistType) {
     const worklistsById = {
       'owned-by-me': 'ownedByMe',
       'for-my-role': 'forMyRole',
       'new-past-day': 'newPastDay',
       'updated-past-three-days': 'pastThree',
       'done-last-thirty-days': 'lastThirty',
     };
+    const types = ['actions', 'flows'];
 
-    if (!worklistsById[worklistId]) Radio.trigger('event-router', 'notFound');
+    if (!worklistsById[worklistId] || !types.includes(worklistType)) Radio.trigger('event-router', 'notFound');
```

Comment:
do we have a polyfill for includes for IE11... just thought of another thing.. above it could be `route: ['worklist/:id/actions', 'worklist/:id/flows']`  and then there's no need to test it.